CoverMyDotNet/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+
+// Information about this assembly is defined by the following attributes.
+// Change them to the values specific to your project.
+
+[assembly: AssemblyTitle ("CoverMyDotNet")]
```

Comment:
Should we consider "CoverMyMeds" for the assembly name?  Our ruby client is named [cover_my_meds](https://rubygems.org/gems/cover_my_meds), so that it is recognizable as ours.